requirements.txt

```diff
@@ -1,8 +1,8 @@
-allennlp>=2.3.0,<=2.9.0.dev20211215
+allennlp==2.10.0
 datasets>=1.13.0,<2.0
 deepdiff>=5.2.0
 jury>=2.1.0,<3.0
 numpy>=1.21.2
 seqeval==1.2.2
```

Comment:
Not related to your changes but I wonder if we can support `>=` in `seqeval` and `tensorboardX` as well?

Reply:
I can check it if you want, but it might take some time. Maybe we can make it an issue and look at it separately?

Reply:
Yes, it would be good as a remainder too even if we do not find time for that now.

Reply:
#57 is created.